eZ/Publish/Core/Repository/SiteAccessAware/SectionService.php

```diff
@@ -75,6 +76,11 @@ public function assignSection(ContentInfo $contentInfo, Section $section)
         return $this->service->assignSection($contentInfo, $section);
     }
 
+    public function assignSectionToSubtree(Location $location, Section $section): void
+    {
+        $this->service->assignSectionToSubtree($location, $section);
+    }
+
     public function deleteSection(Section $section)
```

Comment:
Add docBlock and return a return value typehint

Reply:
the method is not changed in the PR, so unless you want a rabbit hole, we don't do that.

Reply:
Also adding type hints on API is more topic for major versions, next chance to do so is 8.0.

Reply:
> Also adding type hints on API is more topic for major versions, next chance to do so is 8.0.

Why? There's no break for API consumer in this case.

However indeed I agree it might be a rabbit hole, as you've said. But just because it seems out of scope.